public/index.html

```diff
@@ -15,6 +15,7 @@
   <meta property="og:image" content="/img/cover.png">
   <meta name="keywords" content="NeoFS,Neo,Blockchain,智能经济,区块链">
   <meta name="theme-color" content="#ffffff" />
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self' https://verify.walletconnect.com/; script-src 'self'; connect-src 'self' http://127.0.0.1:60004 https://dapi.o3.app:60003 https://rest.t5.fs.neo.org wss://relay.walletconnect.com; style-src 'unsafe-inline' 'self'; font-src 'self'; img-src 'self';">
```

Comment:
Certainly insufficient. rest.fs.neo.org? Other wallets?

Reply:
`rest.fs.neo.org` added, all the wallets works well with this policy string